### PR TITLE
Elixir version of python data preprocessor

### PR DESCRIPTION
**Pull request by Greatlemer**
A rewrite of the python data pre-processor in elixir.

I switched from python to elixir as I found it easier to tackle the problem coming at it this way. I hope that’s ok.

This will actually process a large number of the meeting files and generate a csv with a sanitised version of the data from them, the current output of which is in the src/python/sample_data section of the repo

This should now be in a state that it’s more suitable for pushing to the main branch.

### CODE REVIEW

**Comment by aspiers** [comment]
I can't really review this as I don't have the first clue about Elixir and sadly no time to learn it. But at least we can be confident that it's not breaking any of the other code :)

File: src/elixir/data_sanitiser/test/date_utils_test.exs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      assert normalise_day("31") == 31
+    end
+  end
+
+  defmodule NormaliseMonthTest do
+    use ExUnit.Case
+
+    import DataSanitiser.DateUtils, only: [ normalise_month: 1 ]
+
+    test "Short month names normalise to their integer representation" do
```
Comment (elischutze) [inline src/elixir/data_sanitiser/test/date_utils_test.exs:23]: no may?

Comment (elischutze) [reply src/elixir/data_sanitiser/test/date_utils_test.exs:23]: just saw it's on the next one nvmd

**Review by elischutze — APPROVED** [review]
Does this do all of it or parts of it ? I'll approve it and next time we meet you can walk me through a little bit of it.

File: src/elixir/data_sanitiser/test/date_utils_test.exs (hunk 1 above)
Comment (Greatlemer) [reply src/elixir/data_sanitiser/test/date_utils_test.exs:23]: Yeah, I considered adding it twice but thought it wasn't really a 'shortened' version of the name

**Comment by Greatlemer** [comment]
This is all of it and I'd be happy to talk you through it next time.

I'm currently going through adding in more tests/documentation too (so there'll probably be another PR relating to that in the near future).

**Comment by charlottebrf** [comment]
I know a few people who work on Elixir, @Greatlemer could you do with me doing a reachout?

**Comment by Greatlemer** [comment]
I'll be doing a second PR soon which will have tied up the code quite a bit. It may be useful to have someone to check at that point.

**Comment by charlottebrf** [comment]
Great! Mention me in it so I don't lose the PR in my inbox :)

**Pull request merged**